.huskyrc

```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
```

Comment:
Can we remove this file?

Reply:
This is the config for husky, but we can put this in `package.json` instead. I'm just all for separation of concerns 😉

Reply:
I'm indifferent about using husky, some people don't like it, others like it. But if we are use it, they should be in a separate file (much like our `.babelrc`, `.eslintrc`, etc.)

Reply:
"Gotta keep em separated" - Offspring

Reply:
I agree and will move it back.